packages/kbn-ui-shared-deps/webpack.config.js

```diff
@@ -32,14 +32,22 @@ exports.getWebpackConfig = ({ dev = false } = {}) => ({
   mode: dev ? 'development' : 'production',
   entry: {
     'kbn-ui-shared-deps': './entry.js',
-    'kbn-ui-shared-deps.dark': [
+    'kbn-ui-shared-deps.v7.dark': [
       '@elastic/eui/dist/eui_theme_dark.css',
       '@elastic/charts/dist/theme_only_dark.css',
     ],
-    'kbn-ui-shared-deps.light': [
+    'kbn-ui-shared-deps.v7.light': [
       '@elastic/eui/dist/eui_theme_light.css',
       '@elastic/charts/dist/theme_only_light.css',
     ],
+    'kbn-ui-shared-deps.v8.dark': [
+      '@elastic/eui/dist/eui_theme_amsterdam_dark.css',
```

Comment:
Will amsterdam be the theme for 8.0, or just an alternate theme? I wonder if we want to name the themes by these codenames, or by the major version as you have done here. @snide, thoughts?

Reply:
We talked about ditching the dev name at some point, so I assume we'll drop amsterdam from the filename here and switch this to the new filename once EUI is updated.

Reply:
We will end up switching the "default" theme to the new 8.0 (aka Amsterdam) when 8.0 launches, but keep we might want to keep the old 7 theme around for a bit so users can switch back since it will be quite a major shift. I don't want design to be a blocker to users upgrading. We will of course have the caveat of the old 7 theme no longer being supported aside from major bug fixes.

Reply:
From a branding perspective though, we agreed to call it `v7` and `v8` in the settings right? So we don't need to worry about surfacing the `amsterdam` naming there?

Reply:
Yeah, ultimately the names coming from EUI won't necessarily match the stack anyway, so the abstraction you have here is correct.